package.json

```diff
@@ -26,7 +26,7 @@
     "clean": "make clean",
     "help": "make help",
     "refresh": "make refresh",
-    "test": "make test"
+    "test": "mocha -R list"
```

Comment:
@superkhau is correct in that I recommend running mocha directly from `npm test`.

@bajtos is correct in that it should not be specifying a reporter.

I recommend the test script in package.json simply be `"mocha test/*.test.js"` and that the `Makefile` be used for developer oriented actions so that `npm test` can be reliably used by CI. This will allow CI to better report on the results.